packages/core/test/lib/Kopflos.test.ts.trig

```diff
@@ -25,5 +24,4 @@ ex:FooShape
           .toStream()
       }
     """^^code:EcmaScript ;
-  ] ;
```

Comment:
While working on this and #144, I found a small dissonance I have experienced before.

Which do you think we should have?

1. Directly link implementation to `kl:resourceLoader`
   ```turtle
   [] kopflos:resourceLoader ""^^code:EcmaScript .
   ```
3. Require `code:implementedBy`, like in barnard59
   ```turtle
   [] kopflos:resourceLoader [ 
      code:implementedBy ""^^code:EcmaScript 
   ] .
   ```

Reply:
no strong opinion here